pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -362,6 +362,54 @@ func fixLibvirtCrcNetworkAvailable() error {
 	return nil
 }
 
+func removeLibvirtCrcNetwork() error {
+	logging.Debug("Removing libvirt 'crc' network")
+	_, _, err := crcos.RunWithDefaultLocale("virsh", "--connect", "qemu:///system", "net-info", libvirt.DefaultNetwork)
+	if err != nil {
+		// Ignore since no crc network exists for libvirt
```

Comment:
'since' -> 'if', but why it's more interesting to say why we are ignoring it

Reply:
> why it's more interesting to say why we are ignoring it

Reply:
Why can it happen? `no crc network, user may have removed it themselves, just ignore it`